source_code/main_mcu/src/COMMS/comms_aux_mcu.c

```diff
@@ -243,7 +244,112 @@ void comms_aux_mcu_deal_with_received_event(aux_mcu_message_t* received_message)
             break;
         }
         default: break;
-    }    
+    }
+}
+
+/*! \fn     comms_aux_mcu_handle_FIDO2_auth_cred_msg(FIDO2_message_t const *received_message, aux_mcu_message_t *send_message)
+*   \brief  routine handling authenticating a credential
+*   \param  received_message The received message
+*   \param  send_message The response message
+*   \return FIDO2_MSG_RCVD
+*/
+static comms_msg_rcvd_te comms_aux_mcu_handle_FIDO2_auth_cred_msg(FIDO2_message_t const *received_message, aux_mcu_message_t *send_message)
```

Comment:
any reason for the static given the rest of functions aren't in this file? :)

Reply:
I consider this better style to keep the scope of variables correct. 1) It is only used in this file so it should not be accessible from other files and 2) It makes it easy for someone to look at code and they instantly know that this is only used in this file so they don't have to hunt around to see if something is used somewhere else.